### ISSUE TITLE
Why Conv2D, not Conv1D ?

### ISSUE THREAD

**Issue by littleflow3r** [bentrevett/pytorch-sentiment-analysis#24]
Just wondering, why did you use Conv2d, not Conv1d ?
Thanks.

**Comment by bentrevett** [comment]
The paper I based the implementation off uses 2d convolutions, so I did too. They use 2d convolutions as they experimented with multiple input channels (one from pre-trained word embeddings and one from randomly initialized word embeddings) which you cannot do with Conv1d. 

I also personally found it easy to think about in terms of having the sentence be represented as a 2d object, with words along one axis and embeddings along the other.

**Comment by bentrevett** [comment]
Here's a gist of how to implement the model using Conv1d:
https://gist.github.com/bentrevett/844f5169b86d2d8adc61b93b563a4ddf

Notice how we get pretty much the exact same result.
